reconnect/s3bundle/0.1/config/packages/s3_bundle.yaml

```diff
@@ -0,0 +1,5 @@
+s3_bundle:
```

Comment:
Could you please rename this file to reconnect_s3_bundle.yaml?

Reply:
@Nyholm It has been done, and the bundle updated, thanks